resources/model/OpenStudio.idd

```diff
@@ -31181,13 +31183,98 @@ OS:ZoneHVAC:CoolingPanel:RadiantConvective:Water,
        \minimum 0
        \maximum 1
 
+OS:ZoneHVAC:EvaporativeCoolerUnit,
+       \memo Zone evaporative cooler. Forced-convection cooling-only unit with supply fan,
+       \memo 100% outdoor air supply. Optional relief exhaust node
+       \min-fields 16
+  A1,  \field Handle
+       \type handle
+       \required-field
+  A2,  \field Name
+       \required-field
+       \type alpha
+       \reference ConnectionObject
+  A3,  \field Availability Schedule Name
+       \note Availability schedule name for this system. Schedule value > 0 means the system is available.
+       \note If this field is blank, the system is always available.
+       \type object-list
+       \object-list ScheduleNames
+  A4,  \field Availability Manager List Name
+       \note Enter the name of an AvailabilityManagerAssignmentList object.
+       \type object-list
+       \object-list SystemAvailabilityManagerLists
+  A5,  \field Outdoor Air Inlet Node Name
+       \required-field
+       \type object-list
+       \object-list ConnectionNames
+       \note this is an outdoor air node
+  A6,  \field Cooler Outlet Node Name
+       \required-field
+       \type object-list
+       \object-list ConnectionNames
+       \note this is a zone inlet node
+  A7,  \field Zone Relief Air Node Name
+       \type object-list
+       \object-list ConnectionNames
+       \note this is a zone exhaust node, optional if flow is being balanced elsewhere
+  A8,  \field Supply Air Fan Name
+       \required-field
+       \type object-list
+       \object-list Fans
+  N1,  \field Design Supply Air Flow Rate
+       \required-field
+       \units m3/s
+       \minimum> 0
+       \autosizable
+  A9,  \field Fan Placement
+       \required-field
+       \type choice
+       \key BlowThrough
+       \key DrawThrough
+  A10, \field Cooler Unit Control Method
+       \required-field
+       \type choice
+       \key ZoneTemperatureDeadbandOnOffCycling
+       \key ZoneCoolingLoadOnOffCycling
+       \key ZoneCoolingLoadVariableSpeedFan
+  N2,  \field Throttling Range Temperature Difference
+       \note used for ZoneTemperatureDeadbandOnOffCycling hysteresis range for thermostatic control
+       \type real
+       \units deltaC
+       \required-field
+       \minimum> 0.0
+  N3,  \field Cooling Load Control Threshold Heat Transfer Rate
+       \type real
+       \units W
+       \required-field
+       \note Sign convention is that positive values indicate a cooling load
+       \minimum> 0.0
+  A11, \field First Evaporative Cooler Object Name
+       \required-field
+       \type object-list
+       \object-list EvapCoolerNames
+  A12, \field Second Evaporative Cooler Name
```

Comment:
I know this is just E+. still bothers me that the second doesn't have "Object" in it.
In any case, beware here because GenerateClass.rb will name the getters/setters differently and we don't want that.

I'd remove both "Object Name" from it, so that the getters are `firstEvaporativeCooler` and `secondEvaporativeCooler`